cypress/TestCases/FireboltCertification/Account.feature

```diff
@@ -4,24 +4,24 @@ Feature: Account
    Background: Launch FCA for 'Account'
       Given the environment has been set up for 'Account' tests
       And 3rd party 'certification' app is launched
-
-   @sdk @transport
-   Scenario Outline: Account.<Method> - Positive Scenario: <Scenario>
+#
+   @sdk @transport @Sev0
+   Scenario Outline: <Scenario> Account.<Method>
       When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
-      Then 'Firebolt' platform responds with '<Validation_key>'      
+      Then 'Firebolt' platform responds with '<Validation_key>'
       Examples:
-         | Scenario             | Method | Validation_key | API_Key           |
-         | Validate account ID  | id     | account id     | fetch account ID  | 
-         | Validate account UID | uid    | account uid    | fetch account UID |
+         | Scenario   | Method | Validation_key | API_Key           |
+         | Validate   | id     | account id     | fetch account ID  |
+         | Validate   | uid    | account uid    | fetch account UID |
 
-   @sdk @transport @notSupported @requiresPlatformImplementation
-   Scenario Outline: Account.<method> - Negative Scenario: <Scenario>
+   @sdk @transport @notSupported @requiresPlatformImplementation @Sev2
+   Scenario Outline: <Scenario>
       When '3rd party app' invokes the 'Firebolt' API to '<API_Key>'
       Then 'Firebolt' platform responds with '<Validation_key>'
 
       Examples:
-         | Scenario                              | API_Key                                           | Validation_key                          | method |
-         | Account ID Device not provisioned     | fetch account ID when device is not provisioned   | invalid parameter error for account id  | id     |
-         | Account UID Device not provisioned    | fetch account UID when device is not provisioned  | invalid parameter error for account uid | uid    |
-         | Account ID Account retrieval attempt  | fetch account ID when device is not provisioned  | invalid parameter error for account id  | id     |
-         | Account UID Account retrieval attempt | fetch account UID when device is not provisioned | invalid parameter error for account uid | uid    |
\ No newline at end of file
+         | Scenario                                | API_Key                                           | Validation_key                          |
+         | Account.ID - Device not provisioned     | fetch account ID when device is not provisioned   | invalid parameter error for account id  |
+         | Account.UID - Device not provisioned    | fetch account UID when device is not provisioned  | invalid parameter error for account uid |
+         | Account.ID - Account retrieval attempt  | fetch account ID when account is not provisioned  | invalid parameter error for account id  |
```

Comment:
Im fine with this change, I just want to confirm this is both intentional and that these values exist.